pkg/compiler/debug.go

```diff
@@ -143,6 +147,10 @@ func (c *codegen) emitDebugInfo() *DebugInfo {
 }
 
 func (c *codegen) registerDebugVariable(name string, expr ast.Expr) {
+	if c.scope == nil {
+		// do not save globals for now
```

Comment:
Do we need an issue for it?

Reply:
I think no, because debugger will most likely be changed somehow. In NEO2.0 it used array on the alt.stack together with an index. Now there are 3 separate arrays.
An issue needed for checking our general compatibility with debugger in NEO3 though.

Reply:
#947